plugins/inputs/ssl/README.md

```diff
@@ -0,0 +1,32 @@
+# Check SSL Input Plugin
```

Comment:
So, what exactly should I change? Everything? the directory, file names and the references to them?